e2e/src/runner.c

```diff
@@ -552,7 +552,7 @@ static int cmd_expect_property_set_handler(const struct shell *sh, size_t argc,
     // null variables definition
     char *path = { 0 };
     e2e_byte_array property_value = { 0 };
-    astarte_individual_t individual = { 0 };
+    astarte_data_t individual = { 0 };
```

Comment:
```suggestion
    astarte_data_t data = { 0 };
```

I would also rename the below function `parse_alloc_astarte_invividual`